Modules/ReportGenerator/Resources/views/index.blade.php

```diff
@@ -96,31 +101,14 @@
                         <div id="collapseOne" class="collapse show" aria-labelledby="headingOne" data-parent="#accordion">
                             <div class="card-body">
                                 <p class="note">Frequently used columns</p>
-                                <div class="row">
-                                    <div class="col-sm-3" id="draggable"><p>I am one</p></div>
-                                    <div class="col-sm-3" id="draggable"><p>I am two</p></div>
-                                    <div class="col-sm-3" id="draggable"><p>I am three</p></div>
-                                    <div class="col-sm-3" id="draggable"><p>I am four</p></div>
-                                </div>
-                                <div class="row">
-                                    <div class="col-sm-3" id="draggable"><p>I am five</p></div>
-                                    <div class="col-sm-3" id="draggable"><p>I am six</p></div>
-                                    <div class="col-sm-3" id="draggable"><p>I am seven</p></div>
-                                    <div class="col-sm-3" id="draggable"><p>I am eight</p></div>
-                                </div>
-                                <hr />
-                                <p class="note">All columns that concern reports</p>
-                                <div class="row">
-                                    <div class="col-sm-3" id="draggable"><p>I am one</p></div>
-                                    <div class="col-sm-3" id="draggable"><p>I am two</p></div>
-                                    <div class="col-sm-3" id="draggable"><p>I am three</p></div>
-                                    <div class="col-sm-3" id="draggable"><p>I am four</p></div>
-                                </div>
-                                <div class="row">
-                                    <div class="col-sm-3" id="draggable"><p>I am five</p></div>
-                                    <div class="col-sm-3" id="draggable"><p>I am six</p></div>
-                                    <div class="col-sm-3" id="draggable"><p>I am seven</p></div>
-                                    <div class="col-sm-3" id="draggable"><p>I am eight</p></div>
+                                <div class="row" id="draggable-column">
+                                    @foreach ($draggableComponents as $draggableComponent)
```

Comment:
This is good use of foreach to write compact code. :+1:

Reply:
foreach is indeed a beautiful construct.  One that sometimes leaves me thinking "My code can't work!  There isn't enough....CODE!"  😜

Reply:
My mentors catch everything!!!